plugins/stack-overflow/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@backstage/plugin-stack-overflow",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
```

Comment:
all new packages should have a role declared, as in https://github.com/backstage/backstage/blob/master/plugins/catalog/package.json#L14